app/models/ext_management_system.rb

```diff
@@ -678,6 +678,32 @@ def tenant_identity
     User.super_admin.tap { |u| u.current_group = tenant.default_miq_group }
   end
 
+  def self.output_inventory_status
+    data = includes(:zone)
+           .select(:id, :parent_ems_id, :zone_id, :type, :name, :total_hosts, :total_vms, :total_clusters)
+           .map do |ems|
+             [
+               ems.region_id, ems.zone.name, ems.class.short_token, ems.name,
+               ems.total_clusters, ems.total_hosts, ems.total_vms, ems.total_storages,
+               ems.try(:containers).try(:count)
+             ]
+           end
+    unless data.empty?
+      data = data.sort_by { |e| [e[0], e[1], e[2], e[3]] }
+      # remove 0's (except for the region)
+      data = data.map { |row| row.each_with_index.map { |col, i| i > 0 && col == 0 ? nil : col } }
+      data.unshift(%w(region zone kind ems clusters hosts vms storages containers))
+      # remove columns where all values (except for the header) are blank
```

Comment:
thanks @bdunne - I added specs to exercise this.  yea it is mostly for the container column

ugh - I deleted your comment instead of mine